queries/scala/highlights.scm

```diff
@@ -135,18 +154,29 @@
   "val"
   "var"
   "with"
+  "given"
+  "end"
+  "implicit"
+  "extension"
+  "with"
 ] @keyword
 
 [
   "abstract"
   "final"
-  "implicit"
+  "using"
```

Comment:
Shouldn't this be the same as `given`?

Reply:
Actually yea I believe so. @keynmol was this an oversight by chance?

Reply:
Definitely not intentional

Reply:
Alrighty, updated.